.clang-tidy

```diff
@@ -0,0 +1,4 @@
+---
+Checks: 'readability-*'
```

Comment:
There are more check available, but I'd like to enable them later in separate pull requests as they can change the code semantics.